src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldHaveCorrectPrefix.cs

```diff
@@ -16,7 +16,7 @@ public sealed class IdentifiersShouldHaveCorrectPrefixAnalyzer : DiagnosticAnaly
         public const string RuleId = "CA1715";
 
         private static readonly LocalizableString s_localizableTitle = new LocalizableResourceString(nameof(MicrosoftCodeQualityAnalyzersResources.IdentifiersShouldHaveCorrectPrefixTitle), MicrosoftCodeQualityAnalyzersResources.ResourceManager, typeof(MicrosoftCodeQualityAnalyzersResources));
-        private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(MicrosoftCodeQualityAnalyzersResources.IdentifiersShouldHaveCorrectPrefixTitle), MicrosoftCodeQualityAnalyzersResources.ResourceManager, typeof(MicrosoftCodeQualityAnalyzersResources));
+        private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(MicrosoftCodeQualityAnalyzersResources.IdentifiersShouldHaveCorrectPrefixDescription), MicrosoftCodeQualityAnalyzersResources.ResourceManager, typeof(MicrosoftCodeQualityAnalyzersResources));
```

Comment:
I think this could be another analyzer we write and that checks a match between variable suffix and resource entry suffix.

Reply:
Possible, but that analyzer should possibly be disabled by default, as most analyzer authors do not mandate specific naming suffix. WDYT?

Reply:
I was more thinking about a Roslyn analyzer so something that apply for this repo and maybe roslyn. WDYT?

Reply:
Roslyn repo has a different format for resource strings - it expects the resource name to match the string value with `_` replacing the whitespace character between words. It would apply only to this repo.

Reply:
Let's do nothing for now then.